<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrumCalibrationPointsDialog</class>
 <widget class="QDialog" name="SpectrumCalibrationPointsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectrum Calibration Points</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="CalibrationPointLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="CalibPointOptionsLayout">
       <item>
        <widget class="QLabel" name="calibPointLabel">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Pt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="calibPointText">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="calibPoint">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Calibration point index</string>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="calibPointAddRemoveLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="calibPointAdd">
           <property name="maximumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>190</red>
                 <green>190</green>
                 <blue>190</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
             <family>Liberation Sans</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Add a new calibration point</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="calibPointDel">
           <property name="maximumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>190</red>
                 <green>190</green>
                 <blue>190</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
             <family>Liberation Sans</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Remove current calibration point</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="relativePowerLabel">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Rel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ValueDialZ" name="relativePower" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="toolTip">
          <string>Relative power (dB)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="relativePowerUnits">
         <property name="text">
          <string>dB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="calibratedPowerLabel">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Cal</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ValueDialZ" name="calibratedPower" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="toolTip">
          <string>Calibrated power (dBx)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="calibratedPowerUnits">
         <property name="text">
          <string>dB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="CalibPointPosLayout">
       <item>
        <widget class="QLabel" name="calibPointFrequencyLabel">
         <property name="text">
          <string>F</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ValueDialZ" name="calibPointFrequency" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="toolTip">
          <string>Calibration point frequency (Hz)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="calibPointFrequencyUnits">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>16</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="toolTip">
          <string>Marker frequency (Hz)</string>
         </property>
         <property name="text">
          <string>Hz</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="calibPointsSort">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Sort points by increasing frequency</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources/res.qrc">
           <normaloff>:/sort.png</normaloff>:/sort.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="calibPointDuplicate">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Duplicate current calibration point</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources/res.qrc">
           <normaloff>:/duplicate.png</normaloff>:/duplicate.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="importMarkerZero">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Import data from histogram marker 0</string>
         </property>
         <property name="text">
          <string>M0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="centerFrequency">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Set calibration point frequency to center frequency</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="CalibCommonLayout">
       <item>
        <widget class="QLabel" name="calibInterpModeLabel">
         <property name="text">
          <string>Interp</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="calibInterpMode">
         <property name="toolTip">
          <string>Calibration interpolation mode</string>
         </property>
         <item>
          <property name="text">
           <string>Linear</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Log</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="corrOrSet">
         <property name="minimumSize">
          <size>
           <width>52</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Correct or set value</string>
         </property>
         <property name="text">
          <string>Cor</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ValueDialZ" name="calibrationGlobalCorr" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="toolTip">
          <string>Correction or set (dB)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="calibrationGlobalCorrUnits">
         <property name="text">
          <string>dB</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="globalRelativeCorrection">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Set or apply correction to all relative values</string>
         </property>
         <property name="text">
          <string>Rel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="globalCalibratedCorrection">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Set or apply correction to all calibrated values</string>
         </property>
         <property name="text">
          <string>Cal</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="calibPointsExport">
         <property name="toolTip">
          <string>Export calibration points to .csv file</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources/res.qrc">
           <normaloff>:/export.png</normaloff>:/export.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="calibPointsImport">
         <property name="toolTip">
          <string>Import calibration points from .csv file</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources/res.qrc">
           <normaloff>:/import.png</normaloff>:/import.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValueDialZ</class>
   <extends>QWidget</extends>
   <header>gui/valuedialz.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>calibPointFrequency</tabstop>
  <tabstop>calibPoint</tabstop>
  <tabstop>calibPointAdd</tabstop>
  <tabstop>calibPointDel</tabstop>
  <tabstop>calibPointFrequencyUnits</tabstop>
  <tabstop>importMarkerZero</tabstop>
  <tabstop>centerFrequency</tabstop>
  <tabstop>calibPointsExport</tabstop>
  <tabstop>calibPointsImport</tabstop>
  <tabstop>calibInterpMode</tabstop>
  <tabstop>calibPointDuplicate</tabstop>
  <tabstop>calibrationGlobalCorr</tabstop>
  <tabstop>relativePower</tabstop>
  <tabstop>calibratedPower</tabstop>
  <tabstop>globalRelativeCorrection</tabstop>
  <tabstop>globalCalibratedCorrection</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SpectrumCalibrationPointsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SpectrumCalibrationPointsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
